src/Hl7.Fhir.Base/Model/PocoNodeExtensions.cs

```diff
@@ -161,7 +161,7 @@ private static IEnumerable<PocoNode> parents(this PocoNodeOrList node)
     /// </summary>
     /// <param name="node"></param>
     /// <returns></returns>
-    internal static string? FindFullUrl(this PocoNodeOrList node)
+    public static string? FindFullUrl(this PocoNodeOrList node)
```

Comment:
I can imagine that you need these 2 (FindFullUrl & GetParentResource) to work together for Bundle.entry.fullUrl & ~.resource. But if you call them separately, 
- `parents()` is evaluated twice
- you still have to check whether the parent resource is part of the same entry (it should be, of course, but it is not obvious)
Should we instead offer something like FindParentBundleEntry?